StatusMonitor/Status.cs

```diff
@@ -94,16 +96,22 @@ public enum Flags
         public decimal? longitude;
         public decimal? altitude;
         public decimal? heading;
-        public decimal? fuel;
+        public decimal? fuelInMainTank;
```

Comment:
May have potential for confusing users wrt `main` vs `additional` tanks. Just `fuelInTank` or some variation?

Reply:
I concur. Will change to `fuelInTanks`.